controller/volume_controller.go

```diff
@@ -600,6 +600,14 @@ func (vc *VolumeController) ReconcileVolumeState(v *longhorn.Volume, es map[stri
 		v.Status.InitialRestorationRequired = true
 
 		v.Status.RestoreInitiated = true
+
+		v.Status.Conditions = types.SetCondition(v.Status.Conditions,
```

Comment:
Where do we set the Condition to False when the restore completed?

Reply:
Before we do auto detachment. See the line (line 635) where you put your above comment.

BTW, for DR volumes, the restore condition is always true as long as the volume is not activated, even if the incremental restoration hasn't been triggered. Is this acceptable?

Reply:
It's fine for DR volume. It should be treated as so since it's not usable to the user.

Reply:
I thought the restore complete should trigger detach, not the reverse.

I've filed https://github.com/longhorn/longhorn/issues/1310 , we need to do something about this function after GA.

Reply:
OK. I will move the condition update after the restore complete/activation complete now.